Poke/src/pages/add-user.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useState} from 'react';
+import {SafeAreaView, StyleSheet, ScrollView, View, Text, StatusBar, Alert} from 'react-native';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+import {Input} from '../components/inputs';
+import {SubmitButton} from '../components/submit-button';
+import {addUserValidation} from '../validation';
+
+const AddUser = () => {
```

Comment:
É bom termos o hábito de tipar o que não é trivial para o Typescript:

```suggestion
const AddUser: NavigationPage<Props> = () => {
```

Não lembro de cabeça se é `NavigationPage` mesmo, mas o `react-native-navigation` tem um tipo pra páginas. E as Props, caso não tenha nenhuma customizada, pode ser `{}`